network/src/main/java/bisq/network/p2p/services/peergroup/PersistedPeersStore.java

```diff
@@ -0,0 +1,39 @@
+package bisq.network.p2p.services.peergroup;
+
+import bisq.persistence.PersistableStore;
+import lombok.Getter;
+
+import java.util.Collection;
+import java.util.Set;
+import java.util.concurrent.CopyOnWriteArraySet;
+
+public class PersistedPeersStore implements PersistableStore<PersistedPeersStore> {
```

Comment:
This will require the protobuf implementation. I will post soon a readme about all, then it should be easier to implement it, otherwise just look at some other store like KeyPairStore to see the patterns used.